src/pages/Hosting/HostingComponents/HostingFooter.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const HostingFooter = ({ goToNextStep, goToPrevStep }) => {
+  return (
+    <RightFooter>
+      <Indicator />
+      <GoBackBtn onClick={goToPrevStep}>
+        <span>뒤로</span>
+      </GoBackBtn>
+      <GoNextBtn onClick={goToNextStep}>
+        <span>다음</span>
+      </GoNextBtn>
+    </RightFooter>
+  );
+};
+
+const RightFooter = styled.div`
+  ${props => props.theme.flexBox('row', 'center', 'space-between')}
```

Comment:
- 파라미터자리에서 구조분해 할당을 해 줄 수있습니다.
- flex는 기본 속성으로 'row'를 가지고 있기 때문에 ''(빈 스트링으로 해 주어도 됩니다!)
```suggestion
  ${({theme}) => theme.flexBox('', 'center', 'space-between')}
```

Reply:
처음알았어요!! 적용해주었습니다 :)!!